jsonnet/lib/observatorium-api.libsonnet

```diff
@@ -12,7 +12,7 @@ local defaults = {
   ports: {
     public: 8080,
     internal: 8081,
-    publicGrpc: 8090,
+    'grpc-public': 8090,
```

Comment:
some tools/projects parse the protocol from the port name so sticking to convention.

https://istio.io/latest/docs/ops/configuration/traffic-management/protocol-selection/